api/History.json

```diff
@@ -296,7 +296,7 @@
         },
         "title_parameter": {
```

Comment:
Is there a reason you didn't rename the features themselves?

Reply:
> Is there a reason you didn't rename the features themselves?

Nope, it was just an oversight/miss — thanks for catching that